pkg/server/nfs/nfs_server.go

```diff
@@ -133,3 +145,25 @@ func setRlimitNOFILE(logger logrus.FieldLogger) error {
 	logger.Infof("ending RLIMIT_NOFILE rlimit.Cur %d, rlimit.Max %d", rlimit.Cur, rlimit.Max)
 	return nil
 }
+
+func SetConfigParams(config string) string {
+	var (
+		tmplBuf bytes.Buffer
+		logPath string
+	)
+
+	if os.Getppid() == 1 {
+		logPath = "/proc/1/fd/1"
+	} else {
+		logPath = "/tmp/ganesha.log"
+	}
+
+	tmplVals := struct {
+		LogPath string
+	}{
+		LogPath: logPath,
+	}
+
+	template.Must(template.New("Ganesha_Config").Parse(config)).Execute(&tmplBuf, tmplVals)
+	return tmplBuf.String()
+}
```

Comment:
Add SetConfigParams to determine the log path according to process is LH manager or not.